src/ec/packages/ec-component-menu/ecl-menu.html.twig

```diff
@@ -103,16 +103,16 @@
       </div>
     </a>
   {% if _group == 'group1' %}
-    <div class="ecl-menu-harmonised__site-name">{{ site_name }}</div>
+    <div class="ecl-menu__site-name">{{ site_name }}</div>
   {% endif %}
     <ul
-      class="ecl-menu-harmonised__list"
-      data-ecl-menu-harmonised-list
+      class="ecl-menu__list"
+      data-ecl-menu-list
     >
   {% if items is defined and items is not empty and items is iterable %}
     {% for item in items %}
       {% if item.link is defined and item.link is not empty %}
```

Comment:
We have a problem with the rendered html files that appeared only now with the code introduced in 2.18.0, i guess. prettier breaks on all the rendered files due to false positive, apparently,

> >  SyntaxError: Unexpected closing tag "a". 
> 
> is this a tag:
> <a
>       class="ecl-link ecl-link--standalone ecl-menu__toggle"
>       href={{ toggle_path }}
>       data-ecl-menu-toggle
>     >

The html looks correct but we have to apply some "manual" fix then otherwise this would break php-storybook. So take this as a note for a follow up ticket.. :)

Reply:
ok